### DISCUSSION TITLE
AttributeError: '_Group' object has no attribute 'toscreen'

### DISCUSSION THREAD

**Discussion by OriLiMu** [qtile/qtile#4480] [Q&A] [answered]
I want to make the group stick to screen. 
I follow the steps here: https://docs.qtile.org/en/latest/manual/faq.html#how-can-i-get-my-groups-to-stick-to-screens

Here is my related code : 
```python
    def go_to_group(self, name: str) -> Callable:
        def _inner(lazy) -> None:
            if len(lazy.screens) == 1:
                lazy.groups[name].toscreen()
                return
            if name == 0:
                lazy.focus_screen(1)
                lazy.groups[name].toscreen()
            else:
                lazy.focus_screen(0)
                lazy.groups[name].toscreen()
        return _inner
```

But I get error like this. I check the object _Group which has the method toscreen. 
So how to fix my issue here. 

I output the class of the layz.groups[0] you can check it here. 
`2023-09-28 19:48:50,067 WARNING libqtile keybindings.py:_inner():L129 <class 'libqtile.group._Group'>`

```
2023-09-28 19:48:50,067 WARNING libqtile keybindings.py:_inner():L129 <class 'libqtile.group._Group'>
2023-09-28 19:48:50,076 ERROR libqtile base.py:cmd_function():L216 Exception calling "<function Keybindings.go_to_group.<locals>._inner at 0x7fe428080a40>":
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/libqtile/command/base.py", line 213, in cmd_function
    function(self, *args, **kwargs)
  File "/home/lizhe/.config/qtile/keys/keybindings.py", line 135, in _inner
    lazy.groups[name].toscreen()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: '_Group' object has no attribute 'toscreen'

```

**Comment by elParaguayo** [comment, marked as answer]
I guess you're on v0.22.1. The command name was slightly different in that version. If you look at the docs page for that version: https://docs.qtile.org/en/v0.22.1/manual/faq.html#how-can-i-get-my-groups-to-stick-to-screens you can see the command is `cmd_toscreen`.

Also, it's very confusing that you've changed `qtile` to `lazy` in your code. In qtile, `lazy` is a completely different object.

**Reply by OriLiMu** [reply]
Thanks
